### PR TITLE
Bugfix/multiple ntp and dns

### PR DESCRIPTION
**Pull request by nest1ing**

### CODE REVIEW

File: src/netconfig.cpp [hunk 1]
```diff
@@ -203,15 +203,9 @@ static void cmdDns(Dbus& bus, Arguments& args)
 
     const char* nextArg = args.peek();
     const bool isStatic = nextArg && strcmp(nextArg, "static") == 0;
-    const char* property;
     if (isStatic)
```
Comment (artemsen) [inline src/netconfig.cpp]: I think we may remove ii, it resolves BBMC-313.

**Review by artemsen — APPROVED** [review]

File: src/netconfig.cpp (hunk 1 above)
Comment (nest1ing) [reply src/netconfig.cpp]: As I wrote in the commit message, I left it for backward compatibility.
It does not affect on the command execute any more.
But if you sure that backward compatibility is not required I will remove it completely.

Comment (artemsen) [reply src/netconfig.cpp]: We have no any released firmware with netconfig, so backward compatibility is not required.
Moreover, QA asks to fix the "static" flag: https://j.yadro.com/browse/BBMC-313

Comment (nest1ing) [reply src/netconfig.cpp]: done

**Pull request merged**